### PR TITLE
how to contribute to the web site

### PR DESCRIPTION
**Pull request by szabgab**

### CODE REVIEW

**Comment by sgotti** [comment]
@szabgab Thanks for the PR! I was thinking to move the root directory unders a `site` directory so we could put all these infos in the README.md

**Comment by szabgab** [comment]
Yeah, though a file like this also shows up on the web site so it could be linked from the menu as well.

**Comment by sgotti** [comment]
@szabgab I updated the repository moving everything related to the site contents inside the `site` subdirectory.

New we can add the contributing info in the root `README.md`. Can you please update your PR? Thanks.

**Comment by szabgab** [comment]
rebased and updated.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+```
+npm install
+```
+
+and run the web server locally:
+
+```
+cd site
+../node_modules/vuepress/cli.js dev
```
Comment (sgotti) [inline README.md]: The right command is 

```
vuepress dev site
```

Comment (sgotti) [reply README.md]: sorry I considered using a globally installed vuepress. To use the locally installed vuepress perhaps it'll be better to just add a npm command like

```
"dev": "vuepress dev site"
```

in package.json

and then suggest to use

`npm run dev`

**Comment by sgotti** [comment]
@szabgab LGTM. Thanks a lot! Merging.

**Pull request merged**